proto/spacemesh/mesh.proto

```diff
@@ -20,6 +20,9 @@ service MeshService {
     // Number of layers per epoch (a network parameter)
     rpc EpochNumLayers(google.protobuf.Empty) returns (SimpleInt);
 
+    // Number of transactions per second (a network parameter)
+    rpc TransactionsPerSecond(google.protobuf.Empty) returns (SimpleInt);
```

Comment:
I think that calling this TransactionsPerSecondCapacity or maybe MaxTransactionsPerSecond is better as the current method names sounds like it is the mesh's current processing rate.